subprojects/gdk_rust/src/serialize.rs

```diff
@@ -202,6 +202,12 @@ where
     E: Into<Error>,
     S: Session<E>,
 {
+    // XXX how to handle missing subaccount?
+    let account_num = input["subaccount"]
+        .as_u64()
+        .ok_or_else(|| Error::Other("get_transaction_details: missing subaccount".into()))?
```

Comment:
can we use `Error::InvalidSubaccount`?

Reply:
This particular piece of code was removed in https://github.com/Blockstream/gdk/pull/93/commits/0ff9b55bb3522bcb7b5dbc517c6962f478854766, but there's another similar error message for `get_balance` below it. However, the error variant type is `InvalidSubaccount<u32>`, which usually refers to an invalid account number provided by the user. Maybe create a separate `MissingSubaccount` error type for this?

Reply:
> create a separate MissingSubaccount error type for this

Sounds good for me